### PR TITLE
EDU-414 Reflect OMA through the TestApp

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-414

### CODE REVIEW

File: test-app/src/main.less [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+@import url('./bundles/page-template.less');
 @import url('./bundles/home-page-template.less');
+@import url('./bundles/home-page-illustration.less');
 
-@testapp-primary-color: #0d6868;
+@oma-blue-1: #F2F5F8;
+@oma-blue-2: #4F608E;
+@oma-orange-1: #F5E3D6;
+@oma-orange-2: #EAA58D;
+@oma-primary-color: #6D8BB1;
```
Comment (ahelmberger) [inline test-app/src/main.less:16]: Just a thought: maybe we could use neutral names like `@theme-color-blue` or similar (not having `oma` in the name). That would make copying easier, and we could have only `@edu-...` and non-`@edu-...` (= local) variables. Not a big issue, we can also leave it like it is or discuss at a later point.

File: test-app/src/bundles/home-page-template.less [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+  .m-page-content;
+  padding-bottom: 0;
+  flex: 1 0 0%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: @oma-blue-1;
+
+  @media @edu-media-phone {
+    padding-bottom: 0;
```
[the remark this answers is not in the document]
Comment (adrianaluchian) [reply test-app/src/bundles/home-page-template.less:19]: This is a tricky situation here actually, cause in the `m-page-content` mixin, for mobile we set `padding: calc(@edu-page-content-padding / 2);`. And for this illustration we need to overwrite that for both non-mobile and mobile.

[the remark this answers is not in the document]
Comment (adrianaluchian) [reply test-app/src/bundles/home-page-template.less:19]: Btw, all TestApp files are copied over from OMA without changes (except for the `import` paths, of course)

File: test-app/src/main.less (hunk 1 above)
Comment (adrianaluchian) [reply test-app/src/main.less:16]: With the color palette changes that we do next, these variables will go away and the colors will be directly in educandu. But good point, we should do that in ELMU actually.

**Pull request merged**